.github/workflows/pull-requests.yml

```diff
@@ -10,31 +10,32 @@ on:
 
 jobs:
   build:
-    name: Build
+    name: Test
     runs-on: ubuntu-18.04
     strategy:
       matrix:
         java:
-          - jdk8
-          - jdk11
+          - zulu@1.8.282
```

Comment:
Why not openjdk?

Reply:
I haven't seen an openjdk 8 available but more importantly, openjdk build is slower than zulu or graalvm